README.md

```diff
@@ -159,9 +159,47 @@ The Market Maker Bot configuration had been explained in detail in the recent wo
 }
 ```
 * `mbc_user` describes bot's wallet.
-  * `ur_s_key_path` is the path to signing key file.
-  * `ur_coll` (optional) is the UTxO to be reserved as collateral. Though specifying `ur_coll` is optional but it is advised to set it as then this UTxO would be reserved (i.e., would not be spent) and thus be always available to serve as collateral. It is preferred for `ur_coll` to be pure 5 ADA only UTxO (i.e., no other tokens besides ADA).
   * `ur_stake_address` (optional) is the bech32 stake address (`stake_test1...` for testnet and `stake1...` for mainnet). If specified, bot would place orders at the mangled address so that ADA in those orders (both as an offer or as received payment) would be staked. Note that if an order undergoes partial fill, received payment is in the generated order UTxO and is received by the author of order only when order is completely filled or is cancelled.
+  * `ur_s_key_path` (optional) is the path to payment signing key file (normal or extended). Note that we compute address of the bot (where funds should be provided) using payment key hash from this key and stake key hash from `ur_stake_address` (if provided).
+  * `ur_mnemonic` (optional) is the mnemonic seed phrase to load the wallet. Either one of `ur_s_key_path` or `ur_mnemonic` must be provided. It's sample value is as follows:
```

Comment:
Do we accept words from the English bip-0039 wordlist?

It might be a good idea to refer to this, if this is the case.

I think the official Bitcoin GitHub wordlist is available on GitHub:
https://github.com/bitcoin/bips/blob/master/bip-0039/bip-0039-wordlists.md

Reply:
Yes.